webpack/assets/javascripts/react_app/components/FormAutocomplete/FormAutocomplete.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import AutoComplete from '../AutoComplete';
+import { noop } from '../../common/helpers';
+import './form-autocomplete.scss';
+
+const FormAutocomplete = ({
+  controller,
+  searchQuery,
+  error,
+  results,
+  status,
+  resetData,
+  getResults,
+  initialUpdate,
+  url,
+  isDisabled,
+  data,
+}) => (
+  <div className="form-group form-autocomplete">
+      <label className="col-md-2 control-label" htmlFor="search">Search</label>
```

Comment:
1. not extracted.
2. use Label component?
(afair its solved by using CommonForm)